### PR TITLE
Changed to remove the need for full paths for configs

### PR DESCRIPTION
**Pull request by robhuzzey**
Hey all,

I have made a change that is a breaking one so be careful!

I didn't like how we were required to put the node_modules path in to load the configs from makeup. Mainly because we cannot guarantee the path (see: http://nodejs.org/api/modules.html#modules_loading_from_node_modules_folders).

I've changed so now makeup is a module with one function that simply loads the file required. This way we can change where they are stored later (possibly a document store for example?)

Anyway... please review, thanks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,9 +15,13 @@ $ npm install make-up
...
 
-* [scss-lint](https://github.com/ahmednuaman/grunt-scss-lint) - `config: 'node_modules/make-up/configs/scss-lint.yml'`
-* [jshint](https://github.com/gruntjs/grunt-contrib-jshint) - `config: 'node_modules/make-up/configs/jshint.json'`
-* [jscs](https://github.com/jscs-dev/grunt-jscs) - `config: 'node_modules/make-up/jscsrc.json'`
+```
+var makeup = require( 'make-up' );
+```
+
+* [scss-lint](https://github.com/ahmednuaman/grunt-scss-lint) - `config: makeup( 'scss-lint.yml' )`
+* [jshint](https://github.com/gruntjs/grunt-contrib-jshint) - `config: makeup( 'jshint.json' )`
```
Comment (kevinhodges) [inline README.md]: file is called `jshintrc.json`

File: index.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+var path = require( 'path' );
```
Comment (kevinhodges) [inline index.js]: are we not going 'use strict' with this stuff?

File: index.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+module.exports = function( item ) {
+	return path.resolve( __dirname, './configs/' + item );
```
Comment (kevinhodges) [inline index.js]: Not sure what value `path.resolve` is adding here as you're specifying the exact location anyway.

in my eyes

`return __dirname + '/configs/' + item;`

is the same thing but quicker no?

**Comment by robhuzzey** [comment]
Thanks @kevinhodges Not sure what I was thinking with the "path" module... I had tried something different before I made it simpler, guess I just left that in! :)

**Comment by lukefrake** [comment]
:+1:

**Comment by notABluesSinger** [comment]
:+1:

**Comment by kevinhodges** [comment]
You didn't update the example to contain the correct file name (see my comment)

**Pull request merged**